### ISSUE TITLE
Ethereum 0.8.3 not synching, if I use launch application Ether accounts are gone

### ISSUE THREAD

**Issue by adalparedes** [ethereum/mist#1189]
#### System information
``````
Version: `0.8.3`
OS & Version: Windows 8.1
Node type: `geth
``````

![ether1](https://cloud.githubusercontent.com/assets/7121633/18803337/f52a08a2-81a3-11e6-8404-955654f6bf17.png)

**Comment by EtherTravels** [comment]
I had the same problem. I managed to get my ethereums back by reverting my macOs to the previous day by using time machine. 
The not synching is still not solved and your error messages seem to be the same as mine (I  posted mine here: https://github.com/ethereum/mist/issues/1195)

**Comment by subfactory** [comment]
Then, if you don't have a backup of your ETH you loose it?
An error in the app can make you loose your ETH?

**Comment by evertonfraga** [comment]
@subfactory actually, no. geth won't make any harm to your accounts.

@adalparedes @EtherTravels please check on the old config directory. You'll probably want to copy to the new folder structure.

**Comment by luclu** [comment]
We just published [release `0.8.6`](https://github.com/ethereum/mist/releases/tag/v0.8.6) which will download **geth** `1.4.18` on-the-fly. 
*Please* update to easily roll with the upcoming [Hard Fork](https://blog.ethereum.org/2016/10/13/announcement-imminent-hard-fork-eip150-gas-cost-changes/) on Tuesday.
